meta-intel-edison-bsp/recipes-kernel/linux/files/0050-gpiolib-acpi-fixes-gpf.patch

```diff
@@ -0,0 +1,25 @@
+From 9cdd70bfafd3da81f93ca1173eb36ece3525c3ef Mon Sep 17 00:00:00 2001
+From: Ferry Toth <ftoth@exalondelft.nl>
+Date: Tue, 17 Oct 2023 20:00:31 +0200
+Subject: [PATCH 1/1] gpiolib-acpi: fixes gpf
```

Comment:
Can you take one from upstream?

Reply:
Yes, but this patch was tested on v6.2 - v6.5 and dropped together with those recipies. In v6.6 it is already in upstream and in v6.1 not needed. So, this is only here for historic purposes.